staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -190,9 +191,6 @@ type Plugins struct {
 
 	// PostBind is a list of plugins that should be invoked after a pod is successfully bound.
 	PostBind *PluginSet `json:"postBind,omitempty"`
-
-	// Unreserve is a list of plugins invoked when a pod that was previously reserved is rejected in a later phase.
-	Unreserve *PluginSet `json:"unreserve,omitempty"`
```

Comment:
Isn't v1beta1 a live type? You can't remove this field.

I think you can instead add validation that requires it to either be empty or the same as the reserve list, ignore it everywhere else, and then remove it in the next version.

Reply:
It's not. v1alpha2 was in 1.18. v1beta1 is being introduced now #89701

Reply:
ah ok sorry. I can't keep track of what is where.